### PR TITLE
Adds search to list

### PR DESCRIPTION
**Pull request by devinroche**
## What 
Adds a lil search bar to the CBT List screen to let users search.

image

### CODE REVIEW

**Comment by Flaque** [comment]
SUPER pumped about this!

**Comment by devinroche** [comment]
just added a gif not sure why its **massive** but its there

**Comment by devinroche** [comment]
@Flaque should be done, let me know if it works for you 🎉

**Comment by Flaque** [comment]
@devinroche Awesome! I'll take a look when I get home :)

File: src/CBTListScreen.tsx [hunk 1]
```diff
@@ -111,7 +116,13 @@ const ThoughtItemList = ({
   }
 
   const items = groups.map(group => {
-    const thoughts = group.thoughts.map(thought => (
+    const data = group.thoughts.filter(el => {
+      const itemText = `${el.alternativeThought} ${el.alternativeThought} ${el.challenge}`.toLowerCase()
```
Comment (Flaque) [inline src/CBTListScreen.tsx]: This is really clever. 👌 

But I think you meant to have `automaticThought` for one of them. 😅

File: src/CBTListScreen.tsx [hunk 2]
```diff
@@ -252,8 +266,16 @@ class CBTListScreen extends React.Component<Props, State> {
               </View>
             </Row>
 
+            <TextInput
+              style={{...textInputStyle, marginBottom: 15}}
```
Comment (Flaque) [inline src/CBTListScreen.tsx]: Prefer spacing of margin 18 to match others

**Comment by Flaque** [comment]
See: https://github.com/Flaque/quirk/pull/88#issuecomment-484735056

**Pull request closed without merging**